### PR TITLE
#164798282 Configure Travis CI

### PR DESCRIPTION
**Pull request by anyatibrian**
#### What does this PR do?
Adds travis CI configurations to the project
#### Description of Task to be completed?
- Checks project build
- Creates a test  database 
- Run Test.
#### How should this be manually tested?
- [travis ci](https://travis-ci.org/andela/ah-backend-prime)
#### Any background context you want to provide?
- Travis CI enables us to build and test our project
#### What are the relevant pivotal tracker stories?
[#164798282](https://www.pivotaltracker.com/story/show/164798282)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,14 +6,21 @@ Django==2.1.7
...
 mccabe==0.6.1
 psycopg2==2.7.7
+psycopg2-binary==2.7.7
+pycodestyle==2.5
+
 pycparser==2.19
+pyflakes==2.1.1
 PyJWT==1.7.1
 pylint==2.3.1
+python-decouple==3.1
```
Comment (e-ian) [inline requirements.txt]: @anyatibrian we are no longer using python-decouple as a dependency. please uninstall it

File: requirements.txt [hunk 2]
```diff
@@ -6,14 +6,21 @@ Django==2.1.7
 django-cors-headers==2.5.2
 django-extensions==2.1.6
 djangorestframework==3.9.2
+entrypoints==0.3
```
Comment (e-ian) [inline requirements.txt:9]: @anyatibrian remove any requirements that are not being used in the project at the moment, for instance what does `entrypoints` requirement do at this stage of the project.

**Review by patrickf949 — APPROVED** [review]
Great work keep it up

**Review by e-ian — APPROVED** [review]

**Comment by inno-asiimwe** [comment]
@anyatibrian closing this without merging it because all the work here is redone in `PR-4`.

**Pull request closed without merging**